examples/triton_runner/README.md

```diff
@@ -0,0 +1,217 @@
+# Triton Inference Server integration
+
+BentoML now provides support for Triton Inference Server.
+
+### Why do you want to use Triton?
+
+1. You are current Triton users trying to look for unification among frameworks
+   other than those supported by Triton.
+2. Current BentoML users who want to reduce performance gaps between C++ and
+   Python. While BentoML is rapidly improving, Triton provides better
+   performance under given conditions.
+3. Current Triton users who are looking for reusable pre-processing logic for
```

Comment:
Shall we also add a section like:
"We recommend continuing to use the BentoML runner when:"?